modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2463,6 +2452,28 @@ export class Wallet implements IWallet {
           ],
         });
       }
+
+      case 'Digital Asset': {
+        this.validateNftEntries(sendNftOptions.entries, nftBalance, sendNftOptions);
+
+        const recipients = _.map(sendNftOptions.entries, (entry) => {
+          const tokenData: TokenTransferRecipientParams = {
+            tokenType: TokenType.DIGITAL_ASSET,
+            tokenQuantity: entry.amount.toString(),
+            tokenContractAddress,
+            tokenId: entry.tokenId,
+          };
+          return {
+            address: recipientAddress,
+            amount: '0',
+            tokenData,
+          };
+        });
+        return this.sendMany({
+          ...sendOptions,
+          recipients,
```

Comment:
can the recipients be > 1 ?

Reply:
no, right now there will be just 1 recipient, using a single recipient now.